From 1098f04231502e4f6fc362015672d1399805de21 Mon Sep 17 00:00:00 2001
From: Yanjun Yang <yangyj.ee@gmail.com>
Date: Fri, 14 Jun 2024 08:37:20 +0800
Subject: [PATCH] ARM: stm32: dts: Missing clocks for stm32f429's syscfg.

Without clock definition, SYSCFG will not work, EXTI interrupt for
port other than GPIOA will fail to operate.

Signed-off-by: Yanjun Yang <yangyj.ee@gmail.com>
---
 arch/arm/boot/dts/st/stm32f429.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/st/stm32f429.dtsi b/arch/arm/boot/dts/st/stm32f429.dtsi
index 8efcda9ef8ae..ad91b74ddd0d 100644
--- a/arch/arm/boot/dts/st/stm32f429.dtsi
+++ b/arch/arm/boot/dts/st/stm32f429.dtsi
@@ -579,6 +579,7 @@ spi4: spi@40013400 {
 		syscfg: syscon@40013800 {
 			compatible = "st,stm32-syscfg", "syscon";
 			reg = <0x40013800 0x400>;
+			clocks = <&rcc 0 STM32F4_APB2_CLOCK(SYSCFG)>;
 		};
 
 		exti: interrupt-controller@40013c00 {
-- 
2.45.2

